### ISSUE TITLE
Change is calculated from inconsistent column (flip changes this column)

### ISSUE THREAD

**Issue by mirekphd** [alex/nyt-2020-election-scraper#263]
See below

**Comment by mirekphd** [comment]
A flip seems to be causing the following problem: Change column becomes calculated from Votes Remaining not from Vote Differential.

The states affected by the wrong column choice as a source for Change data are those that flipped and only after the flip has occurred (i.e. GA and PA).

Consult this table for Georgia:
```
4 minutes ago | Biden | 1,584 | 4,602 | 5 | 
12 minutes ago | Biden | 1,585 | 4,607 | ? | 
```
... which should arguably look like this:

```
4 minutes ago | Biden | 1,584 | 4,602 | -1 | 
12 minutes ago | Biden | 1,585 | 4,607 | ? |
```
... or even better like this to avoid confusion as to the basis for computing Change:
```
4 minutes ago | Biden | 1,584 | -1 | 4,602 | 
12 minutes ago | Biden | 1,585 | ? | 4,607 |
```

**Comment by mirekphd** [comment]
A flip seems to be causing this problem. The states affected by the wrong column choice as a source for Change data are those that flipped and only after the flip has occurred (i.e. GA and PA).

**Comment by conerck** [comment]
Change is not meant to show the change in differential, but the size of the block of votes that were included in this update, as explained in the tooltip.

> How many votes were reported in this block?

**Comment by mirekphd** [comment]
> Change is not meant to show the change in differential, but the size of the block of votes that were included in this update, as explained in the tooltip.

So it's based on the wrong column most of the time, and flip repairs it:)

But Change is probably equally interesting for both of these columns / definitions, so why not add it for Votes Differential as well?

**Comment by MrMicrowaveOven** [comment]
Are you suggesting we add a "Change of Votes Differential" column?  That might be interesting, but it would make the tables more complex.  Something we're trying to avoid.

I can see how the word "Change" is a little confusing.  "Votes Counted this Block" would be great, but too long.

**Comment by mirekphd** [comment]
@MrMicrowaveOven : consistency of calculation is the primary reason for opening this Issue. I initially thought Vote Differential is the expected data source for computing Change values, because it was for most states (those that have not flipped). Just look at zeros in the Change values for say Arizona... Only @conerck proved me wrong, pointing out the tool tip for Change column... so let's choose one consistent source for Change column if two separate change columns are deemed excessive complexity.

**Comment by MrMicrowaveOven** [comment]
There is only currently only one source for Change values.  It's just `current_vote_count - vote_count_in_last_iteration`.  This effectively gives us how many votes have happened in this iteration.

The reason why there are some zeros in the Change columns is that sometimes there's an iteration that doesn't involve new votes.  This happens if something else changes in the state, such as a change in the Votes Remaining estimate.  This is still considered an iteration, since something has changed, so a new row is made.  But since this iteration didn't involve new votes, that Change number is zero.

**Comment by Soccertanker** [comment]
The name doesn't have to be long to be more clear. "Votes Counted this Block" is clear, but too long. I think the name should be something simple like "Votes Counted". This is perfectly clear in my opinion because every row is a new block / batch of votes.

**Comment by bojanrajkovic** [comment]
Can someone summarize the requested change here? I see a lot of discussion. Is the consensus that we should change `Change` to `Votes Counted`?

**Comment by mirekphd** [comment]
i> Can someone summarize the requested change here? I see a lot of discussion. Is
> There is only currently only one source for Change values. It's just `current_vote_count - vote_count_in_last_iteration`. 

Sorry, it seems to work, precisely because of the situations you described (where either wrong column seems to be used or non-negativity constraint imposed on top of the correct column), so it doesn't look like it works as intended  (at least to me, and to the OP from issue #306 ). Actually, that other issue - feature request - should probably be preferred, so I'm closing this one here.

**Comment by mirekphd** [comment]
Closing in favour of #306 (please reopen).
